scripts/mkcloud

```diff
@@ -41,6 +40,10 @@ fi
 # causes of this is violation of the common shell coding standard which
 # uses uppercase for environment variables and constants, and lowercase
 # for local variables.
+: ${cinder_backend:=''}
+: ${cinder_netapp_storage_protocol:=iscsi}
+: ${cinder_netapp_login:=openstack}
+: ${cinder_netapp_password:=''}
```

Comment:
If we want to test more backends, the list of cinder_foo_bar variables could get very long. That's my only concern about this PR. But I'm not sure if that is a real problem.